src/jdk.httpserver/share/classes/sun/net/httpserver/simpleserver/resources/simpleserver.properties

```diff
@@ -29,15 +29,17 @@ Usage: java -m jdk.httpserver [-b bind address] [-p port] [-d directory]\n\
 
 options=\
 Options:\n\
--b, --bind-address    - Address to bind to. Default: 0.0.0.0 (all interfaces).\n\
+-b, --bind-address    - Address to bind to. Default: 127.0.0.1 (loopback).\n\
```

Comment:
This assumes that the machine on which the server is run has IPv4 configured. It might not be the case. It can also depend on whether -Djdk.net.preferIPv6Addresses=true is specified on the java command line. The actual address of the loopback (as returned by InetAddress.getLoopackeAddress()) should therefore preferably be passed as parameter to any message that talks about the loopback. It is not such an issue for the wildcard - because AFAIU there's no difference between 0.0.0.0 and ::0 at the OS level